config/initializers/app_settings.rb

```diff
@@ -0,0 +1,24 @@
+class AppSettings
+  def self.load
+    config_file = File.join(Rails.root, "config", "settings.yml")
+    
+    if File.exists?(config_file)
+      all_config = YAML.load(File.read(config_file)) 
+      config = {}
+      ["default", Rails.env].each do |group|
+        config.merge!(all_config[group]) unless all_config[group].blank?
+      end
+
+      config.keys.reject{|k| k=="<<"}.each do |key|
+        cattr_accessor key
+        send("#{key}=", config[key])
+      end
+    end
+  end
+
+  def self.method_missing(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `method_missing(*)` if you want the method to accept any arguments but don't care about them.